Makefile

```diff
@@ -47,7 +47,7 @@ tidy::
 # on windows.
 pin_upstream_sdk: UpstreamPluginSDK=github.com/hashicorp/terraform-plugin-sdk/v2
 pin_upstream_sdk: OurPluginSDK=github.com/pulumi/terraform-plugin-sdk/v2
-pin_upstream_sdk: PluginSDKVersion=v2.0.0-20230912190043-e6d96b3b8f7e
+pin_upstream_sdk: PluginSDKVersion=v2.0.0-20240520223432-0c0bf0d65f10
```

Comment:
Can you raise a PR there as well so the changes can be reviewed? We don't need to merge it

Reply:
I have opened https://github.com/pulumi/terraform-plugin-sdk/pull/39 to track the pulumi/terraform-plugin-sdk change.

It looks like the original issues are resolved. We have tests for resource timeouts:

- https://github.com/pulumi/pulumi-terraform-bridge/blob/a427483773c5ca2d8e9b235952276c63494294bb/pkg/tfbridge/schema_test.go#L695-L696
- https://github.com/pulumi/pulumi-terraform-bridge/blob/a427483773c5ca2d8e9b235952276c63494294bb/pkg/tfbridge/provider_test.go#L3735-L3736
- https://github.com/pulumi/pulumi-terraform-bridge/blob/a427483773c5ca2d8e9b235952276c63494294bb/pkg/tfshim/tfplugin5/provider_test.go#L1069